RecoTauTag/RecoTau/src/AntiElectronIDMVA6.cc

```diff
@@ -44,19 +51,50 @@ AntiElectronIDMVA6<TauType, ElectronType>::AntiElectronIDMVA6(const edm::Paramet
   mvaName_NoEleMatch_wGwoGSF_EC_ = cfg.getParameter<std::string>("mvaName_NoEleMatch_wGwoGSF_EC");
   mvaName_woGwGSF_EC_ = cfg.getParameter<std::string>("mvaName_woGwGSF_EC");
   mvaName_wGwGSF_EC_ = cfg.getParameter<std::string>("mvaName_wGwGSF_EC");
+  mvaName_NoEleMatch_woGwoGSF_VFEC_ = cfg.getParameter<std::string>("mvaName_NoEleMatch_woGwoGSF_VFEC");
+  mvaName_NoEleMatch_wGwoGSF_VFEC_ = cfg.getParameter<std::string>("mvaName_NoEleMatch_wGwoGSF_VFEC");
+  mvaName_woGwGSF_VFEC_ = cfg.getParameter<std::string>("mvaName_woGwGSF_VFEC");
+  mvaName_wGwGSF_VFEC_ = cfg.getParameter<std::string>("mvaName_wGwGSF_VFEC");
 
   usePhiAtEcalEntranceExtrapolation_ = cfg.getParameter<bool>("usePhiAtEcalEntranceExtrapolation");
 
-  Var_NoEleMatch_woGwoGSF_Barrel_ = new float[10];
-  Var_NoEleMatch_wGwoGSF_Barrel_ = new float[18];
-  Var_woGwGSF_Barrel_ = new float[24];
-  Var_wGwGSF_Barrel_ = new float[32];
-  Var_NoEleMatch_woGwoGSF_Endcap_ = new float[9];
-  Var_NoEleMatch_wGwoGSF_Endcap_ = new float[17];
-  Var_woGwGSF_Endcap_ = new float[23];
-  Var_wGwGSF_Endcap_ = new float[31];
-
-  verbosity_ = 0;
+  if (!isPhase2_) {
+    Var_NoEleMatch_woGwoGSF_Barrel_ = new float[10];
```

Comment:
change to unique_ptr, or better yet use std::vectors for a more controlled memory management

Reply:
OK, `float *` (arrays `float[]`) changed to `std::vector<float>` for all `var_`.